### ISSUE TITLE
Stray line on checkbox on IE

### ISSUE THREAD

**Issue by davidread** [alphagov/govuk_elements#518]
A ticked checkbox has a stray hairline. See:
![screen shot 2017-07-10 at 12 57 48](https://user-images.githubusercontent.com/307612/28017188-f519f402-656f-11e7-84b2-1fc8798bcac1.png)

As viewed here:
https://govuk-elements.herokuapp.com/form-elements/#form-checkboxes
on a VM running Windows 7, IE11(Edge) on OSX (Retina). Also seen on IE9 &IE10 emulation (from Edge's dev tools).

**Comment by alex-ju** [comment]
It seems to be a bug caused by the transform rotate (-45deg).
A quick fix might be setting the border to inset or transparent

```
.multiple-choice [type=checkbox] + label::after {
  border-top-style: inset;
}
```
or

```
.multiple-choice [type=checkbox] + label::after {
  border-top-color: transparent;
}
```

**Comment by davidread** [comment]
Thanks @alex-ju both these do it for my project - much appreciated.

I'm afraid I'm not familiar with this repo, nor have my VM set-up to test a local server, so can't provide a PR for this (govuk_elements repo).

**Comment by gemmaleigh** [comment]
Thanks @alex-ju! 💯 

Can you please open a pull request for this fix.

**Comment by nickcolley** [comment]
Reopening until we get a release out

**Comment by gemmaleigh** [comment]
Released in v3.1.1
